gcloud-java-datastore/src/main/java/com/google/cloud/datastore/testing/LocalDatastoreHelper.java

```diff
@@ -218,16 +224,10 @@ public static ProcessStreamReader start(InputStream inputStream) {
     private boolean collectionMode;
     private volatile boolean terminated;
 
-    ProcessErrorStreamReader(InputStream errorStream, String blockUntil) throws IOException {
+    ProcessErrorStreamReader(InputStream errorStream) {
```

Comment:
This is protected stuff so I would't refer to it as API (it's not exposed to the users). Anyway, this changed because older versions of the emulator used to print a ready message to the error stream. Newer versions output the ready message to the standard stream.